.github/workflows/release.yml

```diff
@@ -219,3 +219,21 @@ jobs:
             --amend ghcr.io/tensorchord/vchord-postgres:pg${{ matrix.version }}-v${{ env.SEMVER }}-amd64 \
             --amend ghcr.io/tensorchord/vchord-postgres:pg${{ matrix.version }}-v${{ env.SEMVER }}-arm64
           docker manifest push ghcr.io/tensorchord/vchord-postgres:pg${{ matrix.version }}-v${{ env.SEMVER }}
+
+  publish-pgxn:
+    name: Publish vchord to PGXN
+    runs-on: ubuntu-latest
+    container: pgxn/pgxn-tools
+
+    steps:
+      - name: Checkout Git Repository
+        uses: actions/checkout@v4
+
+      - name: Bundle the Release
+        run: make dist
+      
+      - name: Release on PGXN
+        env:
+          PGXN_USERNAME: ${{ secrets.PGXN_USERNAME }}
+          PGXN_PASSWORD: ${{ secrets.PGXN_PASSWORD }}
+        run: pgxn-release
```

Comment:
It's just a simple script so there is no need to pull a docker image.

```
sudo apt-get install -y libjson-perl

curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/pgxn/docker-pgxn-tools/refs/tags/v1.6.5/bin/pgxn-release | sh
```

You could use this script to upload the dist locally too.

Reply:
> I feel it's more convenient to use the officially provided tools, we don't need to maintain the version.

This does not align with our use case.

```sh
curl --user "tensorchord:${PGXN_PASSWORD}" -F 'submit=Release It!' -F "archive=@./build/vchord--${DISTVERSION}.zip" -H 'X-Requested-With: XMLHttpRequest' --silent --output /dev/stderr --write-out "%{http_code}" https://manager.pgxn.org/upload
```

works, so why we must pull a docker image?